src/server/util/dogstatsd.ts

```diff
@@ -0,0 +1,7 @@
+import { StatsD } from 'hot-shots'
+
+const dogstatsd = new StatsD({
+  prefix: 'go.',
+})
+
```

Comment:
was reading the documentation for `hot-shots`, and they mention the constructor of `StatsD` takes in a `host` parameter. If unspecified, it defaults to check `DD_AGENT_HOST`.

I see that in our elastic beanstalk environment this appears to be defined via `ebextensions`. However, will this pose any problems in local development? 

When `DD_AGENT_HOST` is `undefined`, it assumes the `host` to be `127.0.0.1` port `8125`. There probably won't be any hosts listening on that address/port.

Reply:
there should be no issues on local too since UDP is used by default, but yep it doesn't seem great to send out messages when there's no one listening - will try using the `mock` option on statsd to disable it on dev environment